desktop.py

```diff
@@ -70,7 +80,7 @@ def get_osxcross_sdk(osxcross_bin, arch):
 
 
 def setup_desktop_template(env: dict, opts: DesktopOpts, product: str, target_platform: str, target: str):
-    host_triple = host_triples[target_platform] % target_arch[target_platform][target]
+    host_triple = host_triples[target_platform][target_arch[target_platform][target]]
```

Comment:
Oops, looks like this line broke builds for windows and linux targets. Should be:
```suggestion
    host_triple = host_triples[target_platform][target]
```